### PR TITLE
Locking info link, tooltip, and some variable renames

### LINKED ISSUE #1096: Tooltip for info about locking
Gauge logic is confusing, especially for new users. Add a tooltip and/or docs with the following info:

1) if you have no veSDL, you can lock SDL for some duration
2) if you have veSDL, you can add more SDL which will have the same unlock as your existing lock
3) if you have veSDL, you can increase the unlock time without adding more SDL

Additionally, we should disable inputs to match this logic:
- If the user has veSDL AND they have input into the "lock SDL" field, disable the calendar input
- If the user has veSDL AND they have input into the "date" field, disable the SDL input

### PR DESCRIPTION
**Pull request by ug02fast**
image

closes #1096

### CODE REVIEW

File: src/pages/VeSDL/index.tsx [hunk 1]
```diff
@@ -334,6 +340,9 @@ export default function VeSDL(): JSX.Element {
                 name: "SDL",
                 priceUSD: 0,
               }}
+              disabled={
+                !veSdlTokenBalance.isZero() && Boolean(proposedUnlockDate)
```
Comment (ug02fast) [inline src/pages/VeSDL/index.tsx:344]: @hammeiam @nkrulikovsky can you dbl check this logic?

File: src/pages/VeSDL/index.tsx [hunk 2]
```diff
@@ -347,6 +356,9 @@ export default function VeSDL(): JSX.Element {
                 </Typography>
               </div>
               <DatePicker
+                disabled={
+                  !veSdlTokenBalance.isZero() && sdlTokenInputValueBn.gt(Zero)
```
Comment (ug02fast) [inline src/pages/VeSDL/index.tsx:360]: @hammeiam @nkrulikovsky can you dbl check this logic?

File: src/pages/VeSDL/index.tsx [hunk 3]
```diff
@@ -393,11 +405,29 @@ export default function VeSDL(): JSX.Element {
             >
               {lockedSDLVal.isZero() ? t("createLock") : t("adjustLock")}
             </Button>
-            <Typography textAlign="end">
+            <Box display="flex" justifyContent="space-between">
+              <Tooltip title={<>{t("lockingExplaination")}</>}>
+                <Link
+                  href="https://docs.saddle.finance/" // TODO: link to locking article
```
Comment (ug02fast) [inline src/pages/VeSDL/index.tsx:411]: @hammeiam @nkrulikovsky waiting for docs to publish

File: src/pages/VeSDL/index.tsx (hunk 2 above)
Comment (nkrulikovsky) [reply src/pages/VeSDL/index.tsx:369]: I think date picker can be disable when both `veSdlTokenBalance `and `sdlTokenInputValueBn `are `zero`.

Comment (nkrulikovsky) [reply src/pages/VeSDL/index.tsx:369]: Logic is correct as David suggested.
It prevents user to input SDL amount and Date input simultaneously as @hammeiam suggested.

**Review by nkrulikovsky — APPROVED** [review]
Thank you for fixing my typo 🙏🏻

**Review by hammeiam — APPROVED** [review]
looks great!

File: src/pages/VeSDL/index.tsx [hunk 4]
```diff
@@ -63,10 +65,12 @@ export default function VeSDL(): JSX.Element {
     maxBalance: "",
     sdlTokenInputVal: "",
   })
-  const [veSDLTokenVal, setVeSDLTokenVal] = useState<BigNumber>(Zero)
+  const [veSdlTokenBalance, setVeSdlTokenBalance] = useState<BigNumber>(Zero)
   const [lockedSDLVal, setLockedSDLVal] = useState<BigNumber>(Zero)
   const [unlockConfirmOpen, setUnlockConfirmOpen] = useState<boolean>(false)
-  const sdlTokenValue = parseEther(sdlToken.sdlTokenInputVal.trim() || "0.0")
+  const sdlTokenInputValueBn = parseEther(
+    sdlToken.sdlTokenInputVal.trim() || "0.0",
```
[the remark this answers is not in the document]
Comment (ug02fast) [reply src/pages/VeSDL/index.tsx:72]: I whipped something up. Will keep 👁️ on better solution.

**Pull request merged**